pkgs/applications/office/beancount/beancount-ing-diba.nix

```
@@ -0,0 +1,32 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, python3Packages
+, poetry }:
+
+python3Packages.buildPythonApplication rec {
+  pname = "beancount-ing-diba";
+  version = "0.5.0";
+
+  src = fetchFromGitHub {
+    owner = "siddhantgoel";
+    repo = "beancount-ing-diba";
+    rev = "v${version}";
+    sha256 = "1wzcxkfhv2lqfvrsh7x2bslsav46ybyx96g3058j5f80kllbrsl2";
+  };
+
+  format = "pyproject";
+
+  nativeBuildInputs = [
+    poetry
+  ];
+
+  meta = with lib;
```

Apologies, my previous suggestion wasn't quite right. This should make the derivation evaluate again:
```suggestion
  meta = with lib; {
```

---

Sure. This wouldn't have happened if github would be able to provide a proper editor for these suggestions. Maybe resort to send me patches the next time (via email or just tell me to pull from a publicly reachable remote)! :+1: